crates/mem-pool/src/pool.rs

```diff
@@ -230,6 +205,10 @@ impl MemPool {
         self.mem_block.txs().len().saturating_add(expect_slots) > MAX_MEM_BLOCK_TXS
     }
 
+    pub fn reinject_txs_mut(&mut self) -> &mut Vec<H256> {
```

Comment:
```suggestion
    pub fn pending_restored_tx_hashes(&mut self) -> &mut Vec<H256> {
```

The name should be meaningful